### PR TITLE
read all unread mbox with one click

### PR DESCRIPTION
**Pull request by QvodSoldier**
#### What type of this PR
/kind feature

#### What this PR does / why we need it:
add feature: read all unread mbox with one click

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://terminus-org.app.terminus.io/erda/dop/projects/387/issues/all?id=202613&issueFilter__urlQuery=eyJpdGVyYXRpb25JRHMiOls1MDZdLCJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIiwiUkVTT0xWRUQiXSwiYXNzaWduZWVJRHMiOlsiMTAwMDQzNiJdfQ%3D%3D&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=506&type=REQUIREMENT)

#### Specified Reviewers:

/assign @Effet

### CODE REVIEW

File: modules/core-services/dao/mbox.go [hunk 1]
```diff
@@ -136,3 +136,11 @@ func (client *DBClient) GetMboxByDeduplicateID(orgID int64, deduplicateID, userI
 	}
 	return &mbox, nil
 }
+
+// OneClickRead read all unread mbox with one click
+func (client *DBClient) OneClickRead(orgID int64, userID string) error {
```
Comment (Effet) [inline modules/core-services/dao/mbox.go:141]: `ReadAll(` is better

File: modules/core-services/endpoints/endpoints.go [hunk 2]
```diff
@@ -379,6 +379,7 @@ func (e *Endpoints) Routes() []httpserver.Endpoint {
 		{Path: "/api/mboxs/actions/stats", Method: http.MethodGet, Handler: e.GetMBoxStats},
 		{Path: "/api/mboxs/actions/set-read", Method: http.MethodPost, Handler: e.SetMBoxReadStatus},
 		{Path: "/api/mboxs/{mboxID}", Method: http.MethodGet, Handler: e.GetMBox},
+		{Path: "/api/mboxs/actions/one-click-read", Method: http.MethodPost, Handler: e.OneClickRead},
```
Comment (Effet) [inline modules/core-services/endpoints/endpoints.go:382]: `read-all` is better

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/core-services/dao/mbox.go (hunk 1 above)
Comment (QvodSoldier) [reply modules/core-services/dao/mbox.go]: done

Comment (QvodSoldier) [reply modules/core-services/dao/mbox.go]: done

File: modules/core-services/endpoints/endpoints.go (hunk 2 above)
Comment (QvodSoldier) [reply modules/core-services/endpoints/endpoints.go]: done

**Review by Effet — APPROVED** [review]

**Pull request merged**